Support/Plugins/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+include(OpenFX)
+
+create_ofx_plugin("Basic")
+create_ofx_plugin("Field")
+create_ofx_plugin("Generator")
+create_ofx_plugin("Invert")
+create_ofx_plugin("MultiBundle")
+create_ofx_plugin("Retimer")
+create_ofx_plugin("Tester")
+create_ofx_plugin("Transition")
```

Comment:
For simplicity I'd do this as
```cmake
set(PLUGIN_LIST
    Basic
    Field
    Generator
    Invert
    MultiBundle
    Retimer
    Tester
    Transition
)

foreach(N IN LISTS PLUGIN_LIST)
    create_ofx_plugin($N)
endforeach()
```
Then you can just edit the list to add/remove plugins.